src/main/java/org/prebid/server/bidder/rubicon/proto/RubiconSiteExt.java

```diff
@@ -8,4 +8,9 @@
 public class RubiconSiteExt {
 
     RubiconSiteExtRp rp;
+
+    /**
+     * AMP should be 1 if the request comes from an AMP page, and 0 if not.
```

Comment:
Is it possible this field to be NULL? If yes, pls add something like `or be undefined` to cover all cases.